scripts/dotnet-cli-build/TestTargets.cs

```diff
@@ -89,8 +86,8 @@ public static BuildTargetResult RestoreTestAssetProjects(BuildTargetContext c)
             CleanNuGetTempCache();
 
             var dotnet = DotNetCli.Stage2;
-
-            dotnet.Restore("--verbosity", "verbose", "--disable-parallel", "--infer-runtimes", "--fallbacksource", Dirs.TestPackages)
+            dotnet.Restore("--verbosity", "verbose", "--disable-parallel", "--infer-runtimes",
```

Comment:
as a note, for arg pairs in this case I actually prefer to have them on the same line, makes it slightly more readable IMO